src/Ogooreck/Ogooreck.csproj

```diff
@@ -0,0 +1,12 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+    <PropertyGroup>
+        <TargetFramework>net6.0</TargetFramework>
```

Comment:
As this is going to be a public library, I would suggest something like this...

```
  <PropertyGroup>
    <TargetFramework>net6.0</TargetFramework>
    <ImplicitUsings>enable</ImplicitUsings>
	<PackageProjectUrl>https://github.com/oskardudycz/Ogooreck</PackageProjectUrl>
	<RepositoryUrl>https://github.com/oskardudycz/Ogooreck.git</RepositoryUrl>
	<PublishRepositoryUrl>true</PublishRepositoryUrl>
	<Product>Ogooreck</Product>
	<GenerateDocumentationFile>true</GenerateDocumentationFile>
	<EmbedUntrackedSources>true</EmbedUntrackedSources>
	<AllowedOutputExtensionsInPackageBuildOutputFolder>
		$(AllowedOutputExtensionsInPackageBuildOutputFolder);.pdb
	</AllowedOutputExtensionsInPackageBuildOutputFolder>
	<IncludeSymbols>true</IncludeSymbols>
	<SymbolPackageFormat>snupkg</SymbolPackageFormat>
	<TreatWarningsAsErrors>true</TreatWarningsAsErrors>
	<WarningsAsErrors />
	<LangVersion>10.0</LangVersion>
  </PropertyGroup>
```

Reply:
Added in: https://github.com/oskardudycz/Ogooreck/pull/1/commits/082a3d06dc5e2108dbd8d3d7f1a99dd5f70e3a9d.